volume/volume.go

```diff
@@ -5,9 +5,6 @@ import (
 	"os"
 	"runtime"
 	"strings"
-
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/system"
```

Comment:
Looks like we didn't need this because `system.MkdirAll()` is only called on Linux, in which case it's just an alias for `os.MkdirAll()`; https://github.com/docker/docker/blob/3d13fddd2bc4d679f0eaa68b0be877e5a816ad53/pkg/system/filesys.go#L12-L14